documentation/modules/deploying/proc-exposing-kafka-bridge-service-local-machine.adoc

```diff
@@ -20,15 +20,15 @@ kubectl get pods -o name
 
 pod/kafka-consumer
 # ...
-pod/quickstart-bridge-589d78784d-9jcnr
-pod/strimzi-cluster-operator-76bcf9bc76-8dnfm
+pod/my-bridge-bridge-7cbd55496b-nclrt
+pod/my-cluster-operator-<version>-5886bccd9c-4nsxc
```

Comment:
I don't think the cluster operator pod should be renamed, or? It will be still named `strimzi-cluster-operator-...`.